src/duckling/project/asset.service.ts

```diff
@@ -113,7 +113,16 @@ export class AssetService {
     }
     
     private _getFilePath(assetToLoad : LoadingAsset) : string {
-        let filePath = assetToLoad.filePath || this._store.getState().project.home + "/resources/" + assetToLoad.asset.key + "." + this._fileExtensionFromType(assetToLoad.asset.type);
+        let filePath = "";
+        if (assetToLoad.filePath) {
+            filePath = assetToLoad.filePath;
+        } else {
+            filePath = this._path.join(
+                this._store.getState().project.home,
+                this.resourceFolderName,
+                assetToLoad.asset.key + "." + this._fileExtensionFromType(assetToLoad.asset.type)
```

Comment:
Should this be in the path service as well? Probably unnecessary, since I'm not sure if there's any situation where the ext wouldn't be joined with a dot, but idk

Reply:
fixed